site/content/blog/2021-08-01-whats-new-in-svelte-august-2021.md

```diff
@@ -14,7 +14,6 @@ From The Changelog ([JS Party Ep. 182](https://changelog.com/jsparty/182)) to Sv
 For a full list of features and bug fixes, check out the [Svelte changelog](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md).
```

Comment:
if you'd like to point to the sveltekit changelog in that section it's available at https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md

Reply:
Will do!